### PR TITLE
Add delete-unanswered-surveys-channels admin command

### PR DESCRIPTION
**Pull request by xjahic**
Useful for tidying up channels

### CODE REVIEW

File: src/cogs/admin_commands.py [hunk 1]
```diff
@@ -89,7 +91,42 @@ async def delete_finished_surveys_channels(self, ctx):
...
+        deleted_channels = []
+        logging.info(
+            f"Executing delete_unanswered_surveys_channels command. OLDER_THAN is set to {DELETE_UNANSWERED_SURVEYS_OLDER_THAN}"
+        )
+        unanswered_channel_ids = (
+            db.get_in_progress_survey_channel_ids_created_older_than(
+                DELETE_UNANSWERED_SURVEYS_OLDER_THAN
+            )
+        )
+        for channel_id in unanswered_channel_ids:
```
Comment (gabrielKerekes) [inline src/cogs/admin_commands.py:117]: Perhaps the channel deletions could be extracted into a common function reused in `delete_finished_surveys_channels` e.g. `delete_channels_and_set_survey_status` or simply `delete_survey_channels`.

File: src/db/__init__.py [hunk 2]
```diff
@@ -246,6 +246,11 @@ def get_completed_survey_channel_ids_older_than(minus_interval):
     return [res[0] for res in _fetchall(query)]
 
 
+def get_in_progress_survey_channel_ids_created_older_than(minus_interval):
```
Comment (gabrielKerekes) [inline src/db/__init__.py:249]: Is the `created` part important in the name? I would rather name it `get_in_progress_survey_channel_ids_older_than`. As is actually done with `get_completed_survey_channel_ids_older_than`.

**Review by gabrielKerekes — APPROVED** [review]
Just two small suggestions. I haven't tested it, but I trust that it works 😉 We might want to backup the database before deploying and running it 😄

File: src/db/__init__.py (hunk 2 above)
Comment (xjahic) [reply src/db/__init__.py:249]: Yeah ok, I agree

**Pull request merged**